src/main/java/net/snowflake/ingest/connection/RequestBuilder.java

```diff
@@ -678,12 +678,23 @@ public HttpGet generateHistoryRangeRequest(
    */
   public HttpPost generateStreamingIngestPostRequest(
       String payload, String endPoint, String message) {
-    LOGGER.debug("Generate Snowpipe streaming request: endpoint={}, payload={}", endPoint, payload);
+    final String requestId = UUID.randomUUID().toString();
```

Comment:
My assumption is that the request id will remain the same with retry?

Reply:
thats true!